test/Atmos/EDMF/edmf_model.jl

```diff
@@ -413,6 +413,25 @@ function EDMF(
     return EDMF{FT, N_up, typeof.(args)...}(args...)
 end
 
+Base.@kwdef struct IntrinsicFilter{AbstractFloat, a, w, θ, Q, TKE, cv_θ, cv_Q, cv_θQ} <: AbstractFilterTarget
+    "filter edmf updraft a"
+    a::FT
+    "filter edmf updraft w"
+    w::FT
+    "filter edmf updraft θ"
+    θ::FT
+    "filter edmf updraft Q"
+    Q::FT
+    "filter edmf environment TKE"
+    TKE::FT
+    "filter edmf environment cv_θ"
+    cv_θ::FT
+    "filter edmf environment cv_Q"
+    cv_Q::FT
+    "filter edmf environment cv_θQ"
+    cv_θQ::FT
+end
```

Comment:
These properties need to match the `target` properties above. And `::FT` should probably be `::B` (they're Boolean, right?)

Reply:
you mean AbstractFloat should be replaced by 'Bool' or something similar ?

Reply:
I mean it should look like:
```julia
Base.@kwdef struct IntrinsicFilter{B <: Bool} <: AbstractFilterTarget
    "filter edmf updraft a"
    filter_ρa::B
    "filter edmf updraft w"
    filter_ρaw::B
...
```

Reply:
I changed the names as well the 'filter_' was not adding any information